.travis.yml

```diff
@@ -3,7 +3,9 @@ cache:
   bundler: true
   directories:
     - node_modules
-rvm: 2.1.5
+rvm:
+  - 2.1.6
```

Comment:
Why not leave this at 2.1.5 till we move to 2.2.2? The .ruby-version is still at 2.1.5, better to test with what we're running with.